config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  get '/', to: 'items#index'
+  get '/', to: 'home#home'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.